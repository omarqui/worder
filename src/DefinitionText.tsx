import React, { useState, FunctionComponent }  from 'react';
import "./App.css";

type definitionTextProps = {
    value: string
}

export const DefinitionText = ({value}: definitionTextProps)=> {
   return (
        <div>
            <span>{value}</span>
        </div>
   ) 
}