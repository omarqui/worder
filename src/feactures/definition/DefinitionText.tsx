import React from 'react';
import "../../App.css";

type definitionTextProps = {
    value: string
}

export const DefinitionText = ({value}: definitionTextProps)=> {
   return (
        <div>
            <span>{value}</span>
        </div>
   ) 
}