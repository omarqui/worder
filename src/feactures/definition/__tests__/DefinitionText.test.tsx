import React from "react";
import { render } from "@testing-library/react";
import { DefinitionText } from "../DefinitionText";

it("should DefinitionText match with snapshot", () => {
    const { container } = render(<DefinitionText value="hola" />)
    expect(container).toMatchSnapshot()
})