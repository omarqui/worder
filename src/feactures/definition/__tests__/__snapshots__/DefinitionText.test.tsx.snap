// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should DefinitionText match with snapshot 1`] = `
<div>
  <div>
    <span>
      hola
    </span>
  </div>
</div>
`;
